src/dev/src/app/datagrid/selection-row-mode/selection-row-mode.html

```diff
@@ -35,7 +35,7 @@ <h2>Selection (Row mode)</h2>
 
 <h3>Single selection</h3>
 
-<clr-datagrid [(clrDgSingleSelected)]="singleSelected" [clrDgRowSelection]="true">
```

Comment:
We should leave these in place, revert this file. This feature hasn't been removed so we need to have one place to test it.